Assets/ScriptableObjects/Input/PlayerControls.inputactions

```diff
@@ -40,6 +40,24 @@
                     "processors": "",
                     "interactions": "",
                     "initialStateCheck": true
+                },
+                {
+                    "name": "LeftButton",
+                    "type": "Button",
+                    "id": "96035fad-80ae-4361-a5c4-2a82536d0c16",
+                    "expectedControlType": "Button",
+                    "processors": "",
+                    "interactions": "",
+                    "initialStateCheck": false
+                },
+                {
+                    "name": "RightButton",
```

Comment:
Perhaps these should be called something else than `Left/RightButton`, this is what we could call the `A/D` keybinds that we already have (though in a different way).

Reply:
Renamed to "LeftTab" and "RightTab"